flow/designs/asap7/jpeg_lvt/jpeg_encoder15_7nm.sdc

```diff
@@ -0,0 +1,15 @@
+current_design jpeg_encoder
```

Comment:
Why not symlink this as well?

Reply:
I may want to tighten the clock cycle at some point.